doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -980,7 +980,8 @@ described first.
 
    This command has an effect on unfoldable constants, i.e. on constants
    defined by :cmd:`Definition` or :cmd:`Let` (with an explicit body), or by a command
-   assimilated to a definition such as :cmd:`Fixpoint`, :cmd:`Program Definition`, etc,
+   assimilated to a definition such as :cmd:`Fixpoint`, :cmd:`Definition` with the
+   :attr:`program` attribute, etc,
```

Comment:
In this case, it is not really important to keep the `Program Definition` example. I would just remove it:
```suggestion
   assimilated to a definition such as :cmd:`Fixpoint`, etc.,
```

Reply:
Got it.  "assimilated" is the wrong word, I substituted "associated with a definition".